screens/HomeScreen.tsx

```diff
@@ -323,6 +367,38 @@ const HomeScreen = ({
               }
               keyExtractor={(item, index) => `${index}`}
             />
+            <Spacer />
+            {showBlacklist && (
+              <T center onPress={() => toggleVisibility()}>
+                {isRevealed ? "collapse" : "reveal"} {tokenBlacklist.length}{" "}
+                Hidden Token{tokenBlacklist.length > 1 ? "s" : ""}
```

Comment:
Avoid direct string manipulation like this, prefer replacing the whole word rather than adding a single letter.  
eg:
`Hidden {tokenBlacklist.length > 1 ? "Tokens" : "Token"}`
This is to reduce the cost of adding internationalization later on, it's much easier to swap entire strings by language.  Not an issue right now as we don't have internationalization, but avoiding this pattern now will make it easier when we do.